src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.service.ts

```diff
@@ -31,11 +31,12 @@ export interface FeedbackAffectedStudentDTO {
 }
 export interface FeedbackChannelRequestDTO {
     channel: ChannelDTO;
-    feedbackDetailText: string;
+    feedbackDetailTexts: string[];
+    testCaseName: string;
 }
 @Injectable()
 export class FeedbackAnalysisService extends BaseApiHttpService {
-    search(pageable: SearchTermPageableSearch, options: { exerciseId: number; filters: FilterData }): Promise<FeedbackAnalysisResponse> {
+    search(pageable: SearchTermPageableSearch, levenshtein: boolean, options: { exerciseId: number; filters: FilterData }): Promise<FeedbackAnalysisResponse> {
```

Comment:
```suggestion
    search(pageable: SearchTermPageableSearch, useGrouping: boolean, options: { exerciseId: number; filters: FilterData }): Promise<FeedbackAnalysisResponse> {
```
See overall comment, maybe remove implementation detail "levenshtein" here?